### PR TITLE
Cherry-pick to 6.6: Ignore timestamp in redis, haproxy and system filebeat module (#9855)

### PR DESCRIPTION
**Pull request by kaiyan-sheng**
* Ignore timestamp in redis, haproxy and system filebeat module

* Fixing formatting

(cherry picked from commit b9a20f7ba0b42c98c223a9d34216957a1922372e)

This PR also cherry picked https://github.com/elastic/beats/pull/9506 from commit ea8e6d6c9d5477318733ca058c43ef20efcd5da5

This PR also cherry picked https://github.com/elastic/beats/pull/9862 from commit 9b740a9eecd5f82dc2c35e791ff9651396d299de

### CODE REVIEW

File: filebeat/tests/system/test_modules.py [hunk 1]
```diff
@@ -169,6 +169,7 @@ def _test_expected_events(self, test_file, objects):
 
         for ev in expected:
             found = False
+            clean_keys(ev)
```
Comment (ycombinator) [inline filebeat/tests/system/test_modules.py]: Should we wait till https://github.com/elastic/beats/pull/9855/files#r244956960 is merged and backported, then rebase this PR on `6.x`?

File: NOTICE.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
 Elastic Beats
-Copyright 2014-2018 Elasticsearch BV
+Copyright 2014-2019 Elasticsearch BV
```
Comment (ycombinator) [inline NOTICE.txt]: Should we wait till https://github.com/elastic/beats/pull/9852 is merged and then rebase this PR on `6.x`?

Comment (kaiyan-sheng) [reply NOTICE.txt]: Oh no! Totally missed the comments there. Thanks!!

File: filebeat/tests/system/test_modules.py (hunk 1 above)
Comment (kaiyan-sheng) [reply filebeat/tests/system/test_modules.py]: Oh no! Totally missed the comments there. Thanks!!

**Pull request merged**